nitro-src/claim.ts

```diff
@@ -63,6 +63,11 @@ export function claim(
       };
     }
 
+    surplus = min(
+      surplus,
+      BigNumber.from(guarantees[targetChannelIndex].amount)
+    );
```

Comment:
You're probably refactoring `claim` to be more readable, but I would find this change easier to understand if this value were assigned to a different variable, something like
```suggestion
    const amountToClaim = min(
      surplus,
      BigNumber.from(guarantees[targetChannelIndex].amount)
    );
```

Reply:
WIll take that into account in the refactor.